base_classes/NXtransformations.nxdl.xml

```diff
@@ -55,23 +55,42 @@
 		* ``NX_UNITLESS`` for axes for which no transformation type is specified
 
 		This class will usually contain all axes of a sample stage or goniometer or
-		a detector.  The NeXus default McSTAS coordinate frame is assumed, but additional
-		useful coordinate axes may be defined by using axes for which no transformation
-		type has been specified.
+		a detector. The NeXus default :ref:`McSTAS coordinate frame&lt;Design-CoordinateSystem&gt;`
+		is assumed, but additional useful coordinate axes may be defined by using axes for which
+		no transformation type has been specified.
 
-		The entry point (``depends_on``) will be outside of this class and point to a
-		field in here. Following the chain may also require following ``depends_on``
-		links to transformations outside, for example to a common base table.  If
-		a relative path is given, it is relative to the group enclosing the ``depends_on``
-		specification.
+		**Transformation chain**
+
+		The entry point of a chain of transformations is a field called ``depends_on``
```

Comment:
Somewhere in this document we say depends_on . is the end of the transformation chain but this wording here implies it is the entry point i.e. the start of the transformation chain????